### PR TITLE
Created function to calculated total (codiumAI)

### PR DESCRIPTION
**Pull request by jahel-bytes**
KAN-5

The calculateTotal function is expected to calculate the total cost of items based on their prices and quantities, taking into account discount and shipping options. However, there are several issues with the current implementation.

### CODE REVIEW

**Comment by jahelsantiago** [comment]
@CodiumAI-Agent /review

File: variable-manipulation/main.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+function calculateTotal(items, options) {
+    let t = 0
```
Comment (jahel-bytes) [inline variable-manipulation/main.js:2]: **GPT COMMENT**: 
 - The variable 't' is not descriptive. Consider using a more descriptive name such as 'totalCost'.

File: variable-manipulation/main.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    items.forEach(i => {
+      t += i.price * i.quan
```
Comment (jahel-bytes) [inline variable-manipulation/main.js:4]: **GPT COMMENT**: 
 - The property 'quan' is not clear. If it stands for quantity, consider renaming it to 'quantity' for better readability.

File: variable-manipulation/main.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    })
+    t = t - t * (options.dis || 0)
```
Comment (jahel-bytes) [inline variable-manipulation/main.js:6]: **GPT COMMENT**: 
 - The discount calculation is not clear. If 'options.dis' is the discount rate, it should be divided by 100 to convert it to a decimal.

File: variable-manipulation/main.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+    t = t * 1.1
```
Comment (jahel-bytes) [inline variable-manipulation/main.js:7]: **GPT COMMENT**: 
 - The multiplication by 1.1 on line 7 is not clear. If this is a tax rate, consider making it a constant with a descriptive name.

File: variable-manipulation/main.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+    t = t + (options.ship || 5)
```
Comment (jahel-bytes) [inline variable-manipulation/main.js:8]: **GPT COMMENT**: 
 - The addition of 'options.ship' or 5 is not clear. If 'options.ship' is the shipping cost, consider using a more descriptive name.

**Pull request merged**